_docs/14-animation.md

```diff
@@ -72,6 +74,190 @@ Override this method to specify how the top-level bounds of a component should a
 
 ## Notes
 
-If you implement either method, your component must have a [scope](scopes.html).
+**If you implement either method, your component must have a [scope](scopes.html).**
 
 If updating your component's [state](state.html) changes it's bounds, both `boundsAnimationFromPreviousComponent` and `animationsFromPreviousComponent` will be called. 
+
+## In-depth
+
+### UIKit Animations
+
+Engineers on iOS are used to the simple UIView implicit animations. Here’s what they look like:
```

Comment:
i'd use "block-based" instead of "implicit". AFAICT apple reserves "implicit" for CALayer implicit animations, which are a somewhat different thing (and UIKit disables those entirely I think?)